src/site/stages/build/process-cms-exports/schemas/transformed/node-health_care_region_detail_page.js

```diff
@@ -0,0 +1,60 @@
+module.exports = {
+  type: 'object',
+  properties: {
+    contentModelType: { enum: ['node-health_care_region_detail_page'] },
+    entityType: { enum: ['node'] },
+    entityBundle: { enum: ['health_care_region_detail_page'] },
+    title: { type: 'string' },
+    changed: { type: 'number' },
+    entityUrl: {
+      type: 'object',
+      // TODO: Add breadcrumb here
+      properties: {
+        path: { type: 'string' },
+      },
+      required: ['path'],
+    },
+    fieldAlert: { type: ['string', 'null'] },
+    fieldContentBlock: {
+      type: 'array',
+      items: { $ref: 'Paragraph' },
```

Comment:
Er....not `BlockContent`? :confused: 

Naming is hard...

Reply:
I don't think I understand what you mean.  In the raw schema `field_content_block` is an array of `paragraph` nodes:

![image](https://user-images.githubusercontent.com/2008881/72483306-5d2c1600-37b5-11ea-8f51-7459de67591d.png)

Reply:
Yeah, that's fine. I just think `field_content_block` is an unfortunate name for a block of `paragraph-*`s and not `block_content-*`s. Nothing for you to do.